src/bp/nlu-core/engine.ts

```diff
@@ -64,11 +66,11 @@ export default class Engine implements NLU.Engine {
   }
 
   async train(
+    trainSessionId: string,
```

Comment:
if we had the key contained in the trainSession (which, I believe is in the trainInput, you wouldn't need this)

Reply:
The train session is longer passed to `.train()` and I don't want to add it back. I believe it's only a concern of the nlu module (wrapper) and has nothing to do with the actual nlu core. I don't want to implement a train session in the nlu server, just a key will do the job.

(The TrainSession interface is currently only in the sdk for the studio to use it.)